content/about/governance/_index.md

```diff
@@ -26,98 +18,188 @@ While purpose of the association is to support research software engineering and
 scientific computing, anyone can join as a member. The only requirement is that you
 wish scientific software was more valued, better implemented, more open and FAIR.
 
-An annual membership fee is likely to be necessary to cover costs such as the
+We collect a membership fee to cover costs such as the
 domain name and bank account.
-The member's annual meeting will decide the size of the membership fee.
+The member's annual meeting decides the size of the membership fee.
+
+# Board
+
+The association is governed by a board. In practice, the board will take care of
+the dull day-to-day affairs and, for now, function as the code of conduct
+committee.
+
+The current board members are
+ - Jarno Rantaharju (Aalto University), jarno.rantaharju@aalto.fi
+ - Anne Fouilloux (University of Oslo), annefou@geo.uio.no
+ - Luca Ferranti (University of Vaasa)
+ - Rohit Goswami (University of Iceland)
+ - Samantha Wittke (Aalto University, CSC)
+ - Radovan Bast (UiT The Arctic University of Norway)
+ - Thomas Arildsen (Aalborg University)
+ - Patric Holmvall (Uppsala University)
+
 
 # Statutes
 
-Below is the proposal for the statutes of the association. While the Finnish
-translation will be official (if the association is registered in Finland,)
-the English translation is equivalent.
+The Finnish statutes below are actually official, but the English translation is
+as faithful as possible.
 
 
 ## Statutes of the Association in English
 
 1. Name and domicile of the association:
-    The name of the association is the Nordic Research Software Engineers Ry and it is domiciled in Helsinki, Finland
+    The name of the association is the Nordic Research Software Engineers ry and it is domiciled in Helsinki, Finland
```

Comment:
here ry is with small r, at the beginning of the page it has a capital r

Reply:
It should not be capitalized. I'll check the other places